src/client/common/application/extensions.ts

```diff
@@ -40,6 +41,12 @@ export class Extensions implements IExtensions {
                     }
                 })
                 .filter((item) => item && !item.toLowerCase().startsWith(jupyterExtRoot)) as string[];
+            stacktrace.parse(new Error('Ex')).forEach((item) => {
```

Comment:
The stack trace can also be `at xyx.js:123:12` and this doesn't match the regex.
I've left the regex in case that cases some other edge cases and added the use of the package which should be more accurate and cover the other cases we haven't handled with our regex. (not going to fix the regex, as the package would be more accurate)

FYI - this is a breaking change for AML extension

Reply:
Oops, should remove that `Ex`

Reply:
Not required.